src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -34,6 +34,7 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
         private readonly IResourceNamingService _resourceNamingService;
         private AsyncLazy<RemoteContainerConnectionDetails> _remoteContainerConnectionDetails;
         private readonly IProgress<ProgressUpdate> _progress;
+        private const string ServiceAnnotations = "bridgetokubernetes/ignore-ports";

```

Comment:
nit: check if constants file may be better place for this const. For example similar to: ingress?.Metadata?.Annotations?.ContainsKey(Constants.Https.CertManagerAnnotationKey)